index.html

```diff
@@ -2,10 +2,10 @@
 layout: default
 monetate_page_type: index-main
 meta:
-  title: Crossroads
-  description: No matter what your beliefs, you are welcome here.
-  image:
-    url: "//crds-cms-uploads.imgix.net/content/images/cr-social-sharing-still-bg.jpg?w=1200&h=630&fit=crop"
```

Comment:
Could removing the indentions on this file cause issues?

Reply:
@tcmacdonald Should be fixed now! Great catch.